src/main/java/seedu/address/model/InterviewSchedule.java

```diff
@@ -81,12 +81,11 @@ public void addInterview(Interview interview) {
         interviews.add(interview);
     }
 
-    // Edit function not implemented yet, code commented out to reduce test coverage.
-    /* public void setInterview(Interview target, Interview editedInterview) {
+    public void setInterview(Interview target, Interview editedInterview) {
```

Comment:
Thanks for the hard work on the `Schedule` command 🥲 !!!!!!

Just to check, does the `schedule edit` command disallow interviews to be set in the past - sorry not sure if i missed out the implementation for this.

Reply:
Hey char! No worries, the `parseDateTime` function in `EditScheduleCommandParser| takes care of this issue, just like in `AddScheduleCommandParser`! `parseDateTime` checks for both formatting and whether the input date time is in the past.

Reply:
Ah I see!! :) Okay thanks! 👍